scss/stanford_page.scss

```diff
@@ -0,0 +1,39 @@
+.node-type-stanford-page {
+
+  #main {
+    padding-top: 0;
+    > .container, .span12 {
```

Comment:
Messing with span12 is dirty sauce and is a big no-no to me. You are also setting sizes below within the module without using the grid. That is also a no-no. Override the page template instead of this. You will also want to strip out all of the regions we wouldn't want users putting content in eg: sidebars.